### PR TITLE
BETA: Register googledrivedirect.is-a.dev

### PR DESCRIPTION
**Pull request by c-23404**
Added `googledrivedirect.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by Omxpro — CHANGES_REQUESTED** [review]
invalid CNAME record

File: domains/googledrivedirect.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "c-23404",
+        "email": "mehedims205@gmail.com"
+    },
+    "record": {
+        "CNAME": "c-23404.github.io."
```
Comment (CuteDog5695) [inline domains/googledrivedirect.json:7]: remove the . in "c-23404.github.io."

**Review by CuteDog5695 — CHANGES_REQUESTED** [review]

**Review by Omxpro — APPROVED** [review]

**Comment by Omxpro** [comment]
cannot access website

**Comment by c-23404** [comment]
> cannot access website

what i do?

**Pull request closed without merging**